source/client/index.js

```diff
@@ -26,6 +26,7 @@ module.exports = ({
   routes = ensureRoutes('createClientApp'),
   basepath = '',
   history = createHistory(),
+  shouldUpdateScroll = defaultUpdateScroll,
```

Comment:
I think you can return a position rather than just a bool. So the name is a little funny.